src/ngraph/frontend/onnx_import/op/matmul.cpp

```diff
@@ -114,7 +110,15 @@ namespace ngraph
                                 AxisVector{1, 0},
                                 Shape(right->get_shape().rbegin(), right->get_shape().rend()));
 
-                            return {std::make_shared<ngraph::op::QuantizedDot>(left, right, scale)};
+                            return {ngraph::builder::quantization::QuantizedLinearMatmul(
+                                left,
```

Comment:
I like the "left" and "right" naming convention better than "input_0" and "input_1".

Reply:
input0 and input1 is more like ngraph naming. We've used that at many places including in dot kernels